bogota-abril-2019/css-magazine-cover/sergio_guevara_magazine_cover/html/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" content="Cover of the men's journal magazine">
+    <title>men's journal magazine cover</title>
+    <link rel="stylesheet" type="text/css" href="../css/style.css">
+</head>
+
+<body>
+    
+<div class="magazineBody">
+    
+    <h1>men's journal</h1>
+    
+    <div class= "columns">
+
+        <div class="leftSideSection">
+
+                <h4 class="light">unlikely superhero</h4>
+                <h4 class="light"><strong>paul rudd</strong> is back in <strong>ant-man & the wasp</strong></h4>
```

Comment:
Seems all this text is part of the same context. Should only be one H4 then.

Reply:
I just made the correction, thanks Gabo.